tiles/src/main/java/com/protomaps/basemap/layers/Earth.java

```diff
@@ -17,17 +17,18 @@ public String name() {
 
   public void processOsm(SourceFeature sf, FeatureCollector features) {
     features.polygon(this.name())
-      .setZoomRange(6, 15).setBufferPixels(8);
+        .setAttr("pmap:kind", "earth")
```

Comment:
This is superfluous but nice to have in prep for v4 spec so each feature has a kind. They get merged, so very small file size hit.